### ISSUE TITLE
Optional Nav Bar improvements

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#359]
- Admin and Options should be able to be opened with right arrow or closed with left arrow once highlighted
  - Escape toggle open/closed the menu drawer when a menuitem is highlighted
  - See [W3 ARIA menu pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menu/) for more explicit implementation details
